src/v/cluster/controller_backend.cc

```diff
@@ -327,6 +342,7 @@ controller_backend::bootstrap_ntp(const model::ntp& ntp, deltas_t& deltas) {
               *it);
             it = std::prev(it);
         }
+        vlog(clusterlog.trace, "initial revision source delta - {}", ntp, *it);
```

Comment:
the formatting message needs another `{}`

Reply:
good point, will fix